egs/musdb18/X-UMX/run.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+# Exit on error
+set -e
+set -o pipefail
+
+# Main storage directory.
+# If you start from downloading MUSDB18, you'll need disk space to dump the MUSDB18 and its wav.
+musdb18_dir=data
+
+# After running the recipe a first time, you can run it from stage 1 directly to train new models.
+
+# Path to the python you'll use for the experiment. Defaults to the current python
+# You can run ./utils/prepare_python_env.sh to create a suitable python environment, paste the output here.
+python_path=python
+
+# Example usage
+# ./run.sh --stage 1 --tag my_tag --mix_coef 10.0
+
+# General
+stage=0  # Controls from which stage to start
+tag=  # Controls the directory name associated to the experiment
+# You can ask for several GPUs using id (passed to CUDA_VISIBLE_DEVICES)
+id=$CUDA_VISIBLE_DEVICES
+
+# Data
+sample_rate=44100
+seed=42
+seq_dur=6.0
+samples_per_track=64
+
+# Training
+batch_size=14
+num_workers=4
+optimizer=adam
+lr=0.001
+weight_decay=0.00001
+patience=1000
+lr_decay_patience=80
+lr_decay_gamma=0.3
+epochs=1000
+mix_coef=10.0
+val_dur=80.0
+
+# Evaluation
+eval_use_gpu=-1
+
+. utils/parse_options.sh
+
+if [[ $stage -le  0 ]]; then
+  echo "Stage 0: Downloading MUSDB18 into $musdb18_dir"
```

Comment:
Requirements should be installed at some point.

```suggestion
  echo "Stage 0: Downloading MUSDB18 into $musdb18_dir"
  pip install -r requirements.txt
```